mmv1/third_party/terraform/services/workstations/resource_workstations_workstation_cluster_test.go.tmpl

```diff
@@ -76,6 +78,36 @@ func TestAccWorkstationsWorkstationCluster_Private_update(t *testing.T) {
 	})
 }
 
+func TestAccWorkstationsWorkstationCluster_tags(t *testing.T) {
```

Comment:
This test currently duplicates the example-based test; it should be an [update test](https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test) to ensure that we can patch the tags field (even though we can't read the value.)

(It doesn't actually need to be a new test at all - you can just add modification of the tags field to the existing update test.)